prac_08/dynamic_labels.kv

```diff
@@ -0,0 +1,7 @@
+#Adrianto Hartono
+#14347930
+
+BoxLayout:
+    BoxLayout:
+        orientation:"vertical"
+        id: main
```

Comment:
I think you probably didn't finish

Reply:
Thank you for the feedback, but that is all for the code🙏